app/helpers/requests_helper.rb

```diff
@@ -28,15 +28,20 @@ def select_for_multiple_libraries(form, pickup_libraries)
     return unless pickup_libraries.keys.length > 1
     form.select :destination,
                 pickup_libraries_array(pickup_libraries),
-                label: label_for_pickup_libraries_dropdown(pickup_libraries),
-                selected: 'GREEN'
+                { label: label_for_pickup_libraries_dropdown(pickup_libraries), selected: 'GREEN' },
```

Comment:
I know this wasn't a new addition, but:

`GREEN` is here because it's the default library? Should we move it to a helper or configuration or something?